ipApp/src/Keithley2kDMM.st

```diff
@@ -39,6 +39,7 @@ option +r;
 #define MAX_CHANNELS 22
 #define NUM_FRONT_MODES 15
 #define NUM_REAR_MODES 8
+#define MAX_LENGTH_OF_PV_NAME 61
```

Comment:
It would be better to use the macro `PVNAME_STRINGSZ` from dbDefs.h in Base instead of creating a new macro here. That's a common header file which may already have been included, although I haven't checked.